addons/website_crm_partner_assign/tests/test_partner_assign.py

```diff
@@ -43,63 +43,41 @@ def geo_find(addr, **kw):
         self.startPatcher(patcher)
 
     def test_opportunity_count(self):
-        self.customer_uk.write({
```

Comment:
Isn't it possible to have less changes here ? Especially we have a lot less leads in data, hence seems the test is less reliable ... we should probably compare actual results (e.g. execute the domain and verify the returned ids matches what we expect), instead of comparing domains (in the end we are unsure our domain is always valid)

Reply:
Hello @tde-banana-odoo 
now I have changed the test case and tried to cover all the scenarios in which 
the computation was brokering with multiple leads 
Thanks!! :smiley: